docs/quickstart.md

```diff
@@ -18,11 +18,27 @@ cd postgres-operator
 
 minikube start
 
-# start the operator; may take a few seconds
+# start the operator using one of helm chart or yaml manifests;
+
+# - install postgres-operator with helm chart.
+helm install --name postgres-operator ./charts/postgres-operator
```

Comment:
Doesn't it require also `helm init` to install tiller and stuff on a completely fresh minikube cluster?

Reply:
Ah, I missed that. Added `helm init` instruction.